assets/airports/panc.json

```diff
@@ -0,0 +1,2164 @@
+{
+    "radio": {
+        "twr": "Anchorage Tower",
+        "app": "Anchorage Approach",
+        "dep": "Anchorage Departure"
+    },
+    "icao": "PANC",
+    "iata": "ANC",
+    "magnetic_north": 17.2,
+    "ctr_radius": 80,
+    "ctr_ceiling": 20000,
+    "initial_alt": 20000,
+    "position": ["N61d10m27.02", "W149d59m53.49", "151ft"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N61d10m27.02", "W149d59m53.49"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 360,
+        "speed": 4
+    },
+    "arrivalRunway": "7R",
+    "departureRunway": "33",
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 20000,
+            "airspace_class": "C",
+            "poly": [
+                ["N61.71330056", "W150.2581303"],
+                ["N61.71610417", "W149.7189411"],
+                ["N61.68338250", "W148.9999886"],
+                ["N61.58336778", "W148.8667044"],
+                ["N61.47167000", "W149.0666911"],
+                ["N61.17836167", "W149.3092089"],
+                ["N61.13186222", "W149.3472097"],
+                ["N60.85885528", "W149.5683814"],
+                ["N60.77141583", "W150.2608556"],
+                ["N60.83113250", "W150.5627864"],
+                ["N60.91442417", "W150.8046553"],
+                ["N61.04640556", "W151.2099867"],
+                ["N61.08279000", "W151.2521950"],
+                ["N61.21613278", "W151.2522053"],
+                ["N61.23535694", "W151.2195078"],
+                ["N61.61776556", "W150.5577753"]
+            ]
+        }
+    ],
+    "fixes": {
+        "_RWY25L072TED4DME": ["N61.16874", "W149.82254"],
+        "_RWY25R072TED4DME": ["N61.17059", "W149.82275"],
```

Comment:
These headings _are_ the headings of the runways, so we can probably get away with the more simple `_RWY25L04DME` etc. Your call though.

Reply:
Done 👍 much simpler